src/Classifier.lhs

```diff
@@ -5,6 +5,7 @@ Copyright  Saqib Zardari (c) 2023
 LICENSE: BSD3, see file LICENSE at reasonEq root
 \end{verbatim}
 \begin{code}
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
why this? should be necessary?

Reply:
I think it was needed for the syntax I used for printing the autoLaws